### ISSUE TITLE
Adding support for array values

### ISSUE THREAD

**Issue by levani** [ptarjan/node-cache#102]
I had an use case where I wanted to store array as a value and later push/pop some items.

I think it would make sense to add methods like `push`, `pop`, `unshift`, etc, to work with array values directly from `cache` object, instead of getting the value, manipulating and then putting it back in cache.

I would like to discuss this in more details and send a PR if you think this will be useful in general.

**Comment by CodeLuca** [comment]
^

**Comment by IvanAdmaers** [comment]
Good idea. I’ll think about the possibility of implementing this in my similar package [memory-cache-pro](https://www.npmjs.com/package/memory-cache-pro)
